frontend/src/views/web/partials/dyn-header.html

```diff
@@ -8,7 +8,7 @@
       <ul class="right hide-on-med-and-down">
         <li><a class="waves-effect waves-dark" ui-sref="home" ui-sref-active="active">Home</a></li>
        
-        <li ng-hide="isAuth"><a class="waves-effect waves-dark" ui-sref="auth.login" ui-sref-active="active">LogIn</a></li>
+        <li ng-hide="isAuth"><a class="waves-effect waves-dark" ui-sref="auth.login" ui-sref-active="active" ng-click="auth.resetForm()">LogIn</a></li>
```

Comment:
@Ram81 Don't use click with ui-sref as both are click events and trigger together. So use resetform function on changing of location. Check state route for location based changes.